spec/rubocop/cop/performance/ancestors_include_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Performance::AncestorsInclude do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense when using `ancestors.include?`' do
+    expect_offense(<<~RUBY)
+      Class.ancestors.include?(Kernel)
+            ^^^^^^^^^^^^^^^^^^ Use `<=` instead of `ancestors.include?`.
+    RUBY
+  end
+
+  it 'does not register an offense when using `<=`' do
+    expect_no_offenses(<<~RUBY)
+      Class <= Kernel
+    RUBY
+  end
+
+  it 'corrects `ancestors.include?` to `<=`' do
+    new_source = autocorrect_source('Class.ancestors.include?(Kernel)')
+    expect(new_source).to eq('Class <= Kernel')
```

Comment:
Can you use `expect_correction` instead of `autocorrect_source ... expect.to eq` in after L10?